fanout/join_group_fanout.js

```diff
@@ -0,0 +1,211 @@
+        if (!!args.inputs.length) {
+          return cbk(null, args.inputs);
+        }
+
+        // Exit early when not selecting UTXOs interactively
+        if (!args.is_selecting_utxos) {
+          return cbk(null, []);
+        }
+
+        // Only selecting confirmed utxos is supported
+        const utxos = getUtxos.utxos
+        .filter(n => !!n.confirmation_count)
+        .filter(n => n.address_format !== nestedSegWitAddressFormat);
+
+        // Make sure there are some UTXOs to select
+        if (!utxos.length) {
+          return cbk([400, 'WalletHasZeroConfirmedUtxos']);
+        }
+
+        return args.ask({
+          choices: utxos.map(utxo => ({
+            name: `${asBigUnit(utxo.tokens)} ${asOutpoint(utxo)}`,
+            value: asOutpoint(utxo),
+          })),
+          loop: false,
+          name: 'inputs',
+          type: 'checkbox',
+          validate: input => {
+            // A selection is required
+            if (!input.length) {
+              return false;
+            }
+
+            const tokens = sumOf(input.map(utxo => {
+              return utxos.find(n => asOutpoint(n) === utxo.value).tokens;
+            }));
+
+            const capacity = args.capacity * args.output_count;
+
+            const missingTok = asBigUnit(capacity - tokens);
+
+            if (tokens < capacity) {
+              return `Selected ${asBigUnit(tokens)}, need ${missingTok} more`;
+            }
+
+            return true;
+          }
+        },
+        ({inputs}) => cbk(null, inputs));
+      }],
+
+      // Decode the group invite code and get group details
+      getJoinDetails: ['confirmSigner', 'utxos', ({}, cbk) => {
+        return getJoinDetails({
+          code: args.code,
+          lnd: args.lnd,
+          logger: args.logger,
+          output_count: args.output_count,
+        },
+        cbk);
+      }],
+
+      // Join the Fanout group
+      join: [
+        'confirmSigner',
+        'getMethods',
+        'getJoinDetails',
+        'utxos',
+        ({getJoinDetails, utxos}, cbk) =>
+      {
+        if (getJoinDetails.rate > args.max_rate) {
+          return cbk([
+            400,
+            'ExpectedHigherMaxFeeRateToJoinGroup',
+            {needed_max_fee_rate: getJoinDetails.rate},
+          ]);
+        }
+
+        args.logger.info({waiting_for_other_members: true});
+
+        const join = joinFanout({
+          capacity: getJoinDetails.capacity,
+          coordinator: getJoinDetails.coordinator,
+          count: getJoinDetails.count,
+          id: getJoinDetails.id,
+          inputs: utxos,
+          lnd: args.lnd,
+          output_count: args.output_count,
+          rate: getJoinDetails.rate,
+        });
+
+        join.once('end', ({id}) => cbk(null, {transaction_id: id}));
+        join.once('error', err => cbk(err));
+
+        // After the group is filled the members are matched and peer up
+        join.once('peering', async ({inbound, outbound}) => {
```

Comment:
when i go to joinFanout I don't see peering event existing, is this still actually happening?

Reply:
Hmm, maybe I missed removing this. 

No, there isn't any peering happening other than joiners peering with the coordinator.